### PR TITLE
logging tier ethos-systemd services

### PR DESCRIPTION
**Pull request by ts3ng**
Changelog
Redid https://github.com/adobe-platform/ethos-systemd/pull/59 with cleaner commit history.
adding fleet units for logging hosts splunk and fluentd:
pending worker tier changes
Notes

You must explain how any docker run executions your PR introduces handles log accumulation.

Did you introduce any command that executes docker run?
yes
Are there any dependencies on github.com/adobe-platform/infrastructure?
yes
If so:

Is an update to the base stack required(rare)?
yes
Is an A/B required?
no
Are you dependent on new secrets, or infrastructure in any way?
yes

### CODE REVIEW

**Comment by ts3ng** [comment]
@matthewdfuller cleaned up my PR with better commit history let's go over this one next week.

**Comment by matthewdfuller** [comment]
MUCH cleaner! I'll take a more in-depth look in a bit.

**Comment by ts3ng** [comment]
@matthewdfuller let me know when you get a chance to look at this.  I like to get some feedback early this week so i can get things ready.

File: v1/fleet-local/logging/logging-config@.service [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    -e PROTOCOL_MARATHON="http" \
+    -e POLLER_FREQUENCY=10 \
+    $($IMAGE)"
+
+ExecStop=-/usr/bin/docker stop logging-config
+
+[X-Fleet]
+Global=false
+MachineMetadata=role=logging
+MachineMetadata=role=it-hybrid
```
Comment (matthewdfuller) [inline v1/fleet-local/logging/logging-config@.service]: Can we remove this last line?

**Pull request merged**